.pylintrc

```diff
@@ -133,7 +133,8 @@ disable=print-statement,
         too-few-public-methods,
         too-many-arguments,
         missing-docstring,
-        protected-access
+        protected-access,
+        line-too-long
```

Comment:
Since we didn't seem to have any previous failures for this, did we need to add this one?

Reply:
Removed, but replaced with disabling useless-object-inheritance, as pylint for Python 3 on Travis now complains about classes in the project that use (object) in their definition.